### PR TITLE
T01-Number conversion game

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Input: ");
+         int dec = int.Parse (Console.ReadLine ());
```
Comment (MetaNaveen) [inline Program.cs:7]: If non-numeric value is inputted, kindly assume it as 0. or show an error message.?

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+         String hexa = dec.ToString ("X");
```
Comment (MetaNaveen) [inline Program.cs:8]: Prefer to use smaller case for "string" instead of "String"

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+         Console.WriteLine ($"HEX: {hexa}");
+         string bin = "";
+         while (dec > 0) {
+            int rem = dec % 2;
+            bin = rem + bin;
+            dec = dec / 2;
```
Comment (MetaNaveen) [inline Program.cs:14]: You might like the short hand format "dec /=2".

File: Program.cs [hunk 4]
```diff
@@ -1,7 +1,20 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Input: ");
```
Comment (MetaNaveen) [inline Program.cs:5]: Prompt message could have asked for numeric value.

File: Program.cs [hunk 5]
```diff
@@ -1,7 +1,20 @@
...
+         string decimal_number = Console.ReadLine ();
+         if (int.TryParse (decimal_number, out int dec) && dec!=0) {
```
Comment (MetaNaveen) [inline Program.cs:7]: Isn't 0 a numerical value?

File: Program.cs [hunk 6]
```diff
@@ -1,7 +1,20 @@
...
+            string hexa = dec.ToString ("X");
+            Console.WriteLine ($"HEX: {hexa}");
+            string bin = "";
+            while (dec > 0) {
```
Comment (MetaNaveen) [inline Program.cs:11]: Does it support negative values because I see no value for binary if I enter negative value?

File: Program.cs [hunk 7]
```diff
@@ -1,7 +1,20 @@
...
+               int rem = dec % 2;
+               bin = rem + bin;
```
Comment (MetaNaveen) [inline Program.cs:13]: Since "rem" is not used only in one place, it could've been used straight away in the "bin" assignment line.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 4 above)
Comment (Priyanka15-12) [reply Program.cs]: Done

File: Program.cs (hunk 6 above)
Comment (Priyanka15-12) [reply Program.cs]: Changed

File: Program.cs (hunk 7 above)
Comment (Priyanka15-12) [reply Program.cs]: Remainder used as expression.

**Comment by MetaNaveen** [comment]
Working as expected.

**Final notes:**
_using System;_ is used twice.
Space between _Operator and Operands_ should be maintained

**Comment by Priyanka15-12** [comment]
Formatting issue fixed.

**Pull request closed without merging**